### ISSUE TITLE
why loss is nan ? 

### ISSUE THREAD

**Issue by centurions** [sirius-ai/LPRNet_Pytorch#21]

**Comment by sirius-ai** [comment]
1) Please try to use little learning rate firstly.
2) Please check your training datasets whether have unclean data in it.
